### ISSUE TITLE
"Editor" class has a nested relationship with "composition"

### ISSUE THREAD

**Issue by Wec4Dre96** [we-miks/collaborative-editor#9]
![image](https://user-images.githubusercontent.com/43702793/117528730-417dfc00-b006-11eb-8280-7da41196fe50.png)

**Comment by lencyforce** [comment]
Yes. That's because those 2 objects need to send messages to each other. Of course there're better designs to avoid this, but right now I don't really have time for this. PR is welcome. Lol.
